src/Lean/Widget/InteractiveGoal.lean

```diff
@@ -13,7 +13,10 @@ namespace Lean.Widget
 open Server
 
 structure InteractiveHypothesis where
```

Comment:
Would something like `InteractiveHypothesisBundle` be a better name?

Reply:
fixed